### ISSUE TITLE
Make an Accessibility Guide to the Space

### ISSUE THREAD

**Issue by ibrand** [ProgressiveHackNight/admin#20]
We should have a guide to the space for accessibility purposes.

@laveshaparker thoughts on more description for this?

**Comment by laveshaparker** [comment]
@ibrand I think that description is fitting! I'll assign it to myself :)
